### ISSUE TITLE
production: use default key if no translation available 

### ISSUE THREAD

**Issue by bagage** [olivierperez/o80-i18n#7]
Instead of returning `[missing key: $KEY]`, at least in production code, could it be possible to return $KEY instead? 
In case of missing translations, having mixed languages is (I believe) OK and quite common but having some debug statements instead is quite unnatural for end users.

**Comment by olivierperez** [comment]
Hi,

Having an ugly text when there is no translation is a choice. At development time, the developers/testers clearly see that there is something missing. If we replace it by the key, the user may think it is the right word when it is not.

**Comment by bagage** [comment]
Absolutely, and I do agree with that. But I am talking about production code here, where some translations might be missing anyway (because you cannot _force_ translators to do _all_ translations before going to production).

**Comment by olivierperez** [comment]
Actually, you can tell translators to translate all keys :smiley:, by not release an not-translated language by example.

By the way, I understand your position. If you want to propose a Pull Resquest for this feature it's OK.

It should be disabled by default, and have to be configurable like this :
- original: MISSING KEY...
- by word: The same word that is display for every langs
- by key: The word will be found at a defined key in the current locale

Are you fine with it?

**Comment by bagage** [comment]
What do you mean by "by word" and by "by key"?
For instance `__("Comment", "Add another comment")` should return, for French:
- if translation is present: `Ajouter un autre commentaire`
- original: `[missing key: Comment.Add another comment]` (the current one)
- by word: `Add another comment`?
- by key: another sentence picked from French keys? What's the purpose of that? What if that another key is also missing?

**Comment by olivierperez** [comment]
Yes for the "by key", "another sentence picked from French keys" is what I meant.

Actually it was just an idea. I prefer the mode "original" for my projects.
